lib/docassemble/docassemble_api_interface.js

```diff
@@ -329,29 +329,6 @@ da_i.pull = async function ( pull_options ) {
   }
 };  // Ends da_i.pull()
 
-
-da_i.install_on_server = async function ( install_options ) {
```

Comment:
I think it's still useful to have these APIs as a part of ALKiln. I know we aren't using them directly in this PR, but at some point you had asked me how to make things more portable so others could run it, and this is how; the only reason that we switched to `dainstall` is because DA has a bug with weird workarounds for handling packages where the repo name is different than the package contents.